src/docfx/config/ops/OpsCredentialHandler.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+
+namespace Microsoft.Docs.Build
+{
+    internal class OpsCredentialHandler : DelegatingHandler
```

Comment:
>OpsCredentialHandler [](start = 19, length = 20)

should `OpsCredentialHandler` be a `CredentialHandler`?

Reply:
They handling different kind of credentials, opsCredentialHanlder doesn't inherit from the `CredentialHandler`, Rename `CredentialHandler` may avoid the confusion, but I don't have a better idea